### PR TITLE
Adds the image count assessment

### LINKED ISSUE #357: Create assessment for imageCount
Create an assessment for imageCount

### PR DESCRIPTION
**Pull request by jcomack**
This adds the image count assessment to YoastSEO.js

Fixes #357

### CODE REVIEW

File: js/assessments/imageCount.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	return {};
+};
+
+/**
+ * Calculate the score based on the current image alt-tag count.
+ * @param {object} altProperties An object containing the various alt-tags.
+ * @param {object} i18n The locale object.
+ * @returns {object} The resulting score object.
+ */
+var calculateAltTagCountResult = function( altProperties, i18n ) {
```
Comment (atimmer) [inline js/assessments/imageCount.js]: Why don't you call this something like `assessImages`?

File: js/assessments/imageCount.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+};
+
+/**
+ * Execute the Assessment and return a result.
+ * @param {Paper} paper The Paper object to assess.
+ * @param {Researcher} researcher The Researcher object containing all available researches.
+ * @param {object} i18n The locale object.
+ * @returns {AssessmentResult} The result of the assessment, containing both a score and a descriptive text.
+ */
+var imageCountAssessment = function( paper, researcher, i18n ) {
```
Comment (atimmer) [inline js/assessments/imageCount.js]: Isn't this just an `imagesAssessment`, it is a value judgement about the images inside the text.

**Comment by atimmer** [comment]
CR Done 👍🏻, two remarks about naming, but that shouldn't stop this PR.

**Comment by terw-dan** [comment]
Acceptance :+1:

**Pull request merged**